### ISSUE TITLE
Create a "deepdream" video.fx

### ISSUE THREAD

**Issue by mgaitan** [Zulko/moviepy#184]
Few days ago, Google published a [blog post](http://googleresearch.blogspot.ch/2015/06/inceptionism-going-deeper-into-neural.html) and a [related code](https://github.com/google/deepdream/blob/master/dream.ipynb) (in python) describing a technique to use neural networks to do the opposite of what they are frequently used for: instead to classify a given image, use the information the network has about a classifier and a seed (potentially just noise) to create a new image. a truly artificial dream! 

the "deepdream" turns a trending topic around the "nerd" social web, because it's really fun. A very [active community](www.reddit.com/r/deepdream/) arose in reddit and there are people extending the concept to related fields, [like video](https://github.com/graphific/DeepDreamVideo) (python implementation)

As Moviepy is the standard-bearer library for python geeks who like to play (for fun and profit) with video, and already has all the helpers needed to "apply" the algorithm, would be great to adapt those codes to create an easy to use video fx function.

**Comment by Zulko** [comment]
Definitely. I have been wanting to do that in the last two weeks but I really haven't had the time. And I crashed my ubuntu windows manager trying to install nvidia drivers to cuda-accelerate the DeepDreaming. But yes it would do nicely with MoviePy ! (maybe not as something part of the library, more as a tutorial+example kind of blog post).

**Comment by OsaAjani** [comment]
Thank you for your contributions and for reporting issues in this repository. With the release of v2, which introduces significant changes to the codebase and API, we’ve reviewed the backlog of open PRs and issues. Due to the length of the backlog and the likelihood that many of these are either fixed or no longer applicable, we’ve made the decision to close all previous PRs and issues.

If you believe that any of these are still relevant to the current version or if you'd like to reopen a related discussion, please feel free to create a new issue or pull request, referencing the old one.

Thank you for your understanding and continued support!
